packages/contracts-bedrock/src/L2/ISuperchainERC20.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/// @title ISuperchainERC20Extensions
+/// @notice Interface for the extensions to the ERC20 standard that are used by SuperchainERC20.
+///         exists in case developers are already importing the ERC20 interface separately and
+///         importing the full SuperchainERC20 interface would cause conflicting imports.
+interface ISuperchainERC20Extensions {
+    /// @notice Emitted when tokens are sent from one chain to another.
+    /// @param _from    Address of the sender.
+    /// @param _to      Address of the recipient.
+    /// @param _amount  Number of tokens sent.
+    /// @param _chainId Chain ID of the recipient.
+    event SendERC20(address indexed _from, address indexed _to, uint256 _amount, uint256 _chainId);
+
+    /// @notice Emitted when token sends are relayed to this chain.
+    /// @param _to     Address of the recipient.
+    /// @param _amount Number of tokens sent.
+    event RelayERC20(address indexed _to, uint256 _amount);
```

Comment:
Hmmm... does seem like this should be part of the spec. cc @tynes

Reply:
I am not super opposed here, this information will exist in the event emitted by the `L2ToL2CrossDomainMessenger` but it will be abi encoded, the data people would like this information

Reply:
In the past we've had some small hurdles getting data due to missing events/params, so I would lean towards including more data in an event to simplify data fetching. If the relay methods end up being too expensive we can address that separately as there are probably higher leverage ways to reduce cost than cutting out event params

Reply:
We did update the spec, will follow up with a fix